nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/util/hive/HiveConfigurator.java

```diff
@@ -74,7 +74,8 @@ public HiveConf getConfigurationFromFiles(final String configFiles) {
 
     public void preload(Configuration configuration) {
         try {
-            FileSystem.get(configuration);
+            FileSystem.get(configuration).close();
```

Comment:
Did you test this with respect to why this line was added in the first place (High Availability clusters I think)? Just wanted to make sure it wouldn't introduce a regression.